src/main/stackpack/stackpack.conf

```diff
@@ -19,6 +19,13 @@ configurationUrls {
 }
 faqs = []
 steps = [
+  {
+    name = "splunk_instance_name"
+    display = "Splunk Instance Name"
```

Comment:
Can we also add some text to the Markdown file(s) describing what this field does?
I think we also did that for another StackPack

Reply:
Added in the configuration.md file.